src/client/css/partials/floatingBanner.css

```diff
@@ -0,0 +1,54 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+.floating-banner {
+  align-items: center;
+  background: var(--background-gradient);
+  bottom: 0;
+  color: #fff;
+  box-shadow: 0 -6px 6px -8px #000;
+  gap: var(--padding-md);
+  justify-content: center;
+  font-size: 0.875rem;
+  left: 0;
+  padding: var(--padding-sm) 10vw;
+  position: sticky;
+  width: 100%;
+  z-index: 2;
```

Comment:
Added a comment. Actually an index of `1` is enough because: [736789d](https://github.com/mozilla/blurts-server/pull/3067/commits/736789d647e6bb0255ff53dc34f83cc2edcee564).